GoogleDataTransport/GDTCORTests/Unit/GDTCOREventTest.m

```diff
@@ -67,11 +67,60 @@ - (void)testArchiving {
   }
   XCTAssertEqualObjects(decodedEvent.mappingID, @"testID");
   XCTAssertEqual(decodedEvent.target, 42);
-  XCTAssertEqualObjects(decodedEvent.dataObjectTransportBytes,
-                        [@"someData" dataUsingEncoding:NSUTF8StringEncoding]);
+  event.dataObject = [[GDTCORDataObjectTesterSimple alloc] initWithString:@"someData"];
   XCTAssertEqual(decodedEvent.qosTier, GDTCOREventQoSTelemetry);
   XCTAssertEqual(decodedEvent.clockSnapshot.timeMillis, timeMillis);
   XCTAssertEqual(decodedEvent.clockSnapshot.timezoneOffsetSeconds, timezoneOffsetSeconds);
 }
 
+/** Tests testSettingVariables. */
```

Comment:
This comment should be a description of what the test is testing, not the test's name.